web/app.php

```diff
@@ -18,10 +18,20 @@
         die;
     }
 
+    if (!empty($_GET['FEATURE_DIGEST_CHANNEL'])) {
+        setcookie('FEATURE_DIGEST_CHANNEL', $_GET['FEATURE_DIGEST_CHANNEL']);
```

Comment:
are all the `setcookie` defaults like duration, path, host and ssl enforcement ok for this use case?

Reply:
Yep.